internal/state/relationship/capturer.go

```diff
@@ -135,26 +135,22 @@ func (c *CapturerImpl) decrementRefCount(svcName types.NamespacedName) {
 	}
 }
 
-// FIXME(pleshakov): for now, we only support a single backend reference
 func getBackendServiceNamesFromRoute(hr *v1beta1.HTTPRoute) map[types.NamespacedName]struct{} {
 	svcNames := make(map[types.NamespacedName]struct{})
 
 	for _, rule := range hr.Spec.Rules {
-		if len(rule.BackendRefs) == 0 {
-			continue
-		}
-		ref := rule.BackendRefs[0].BackendRef
+		for _, ref := range rule.BackendRefs {
+			if ref.Kind != nil && *ref.Kind != "Service" {
+				continue
```

Comment:
I feel we're missing logging here.

If a user configures two backends, one a Route and one a Service. How do they know the Route isn't accepted?

Reply:
This check is only to track relationships b/w routes and services. We will report the misconfiguration once we build the graph.

Reply:
We log something in validateBackendRef. Will the above scenario get logged there?

Reply:
Yes, we will log a warning.